zmon-controller-app/src/main/java/org/zalando/zmon/config/EumTracingProperties.java

```diff
@@ -0,0 +1,26 @@
+package org.zalando.zmon.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "opentracing.eum")
+public class EumTracingProperties {
+
+    public String zmonConfig;
```

Comment:
Frankly speaking, using just `String` is error-prone. If it's a nested map provided by configuration JSON, then it can be malformed (e.g. missing `"` or `,`), but you never know that until you stop receiving traces because of broken configuration.
I'd rather define some type here. If it's generic multi-level dictionary object, then I'd consider using `Map<String, Object>` type. In this case if JSON config is broken, you 'll see it when application starts. Also it helps to understand what kind of configuration is expected.